ls1intum/Artemis
# [Exam] Client: Summary

### Checklist

### Motivation and Context


Closes issue #1569 
### Description

- implemented exam summary component for all exercise types
  - used collapsable exercise cards for each submission
  - added title and problem statement of each submission/exercise
  - added students submission

### Steps for Testing

1. Create an exam with different exercise types (currently only quiz, text and programming exercises are supported by exam mode)
1.1 add yourself (or any student) to the exam
1.2 make sure to click generate individual exams + prepare exercise start
2. Participate in that exam
3. Wait until exam is over
4. Summary Page should appear
4.1 Check whether your submission, the title and the problem statement for a text and/or programming exercises are shown correctly 
4.2 Check whether editing the exercises is disabled

### Screenshots


Text Exercise:
![image](https://user-images.githubusercontent.com/50466283/85931968-6b37a400-b8c8-11ea-83f0-6dba73e17eba.png)
Quiz Exercise: 
![image](https://user-images.githubusercontent.com/50466283/85931992-ac2fb880-b8c8-11ea-9ae8-e3b223f33bac.png)

Edit: Current bug with the programming exercises will prevent you from generating this specific exercise type
